app/qml/Highlight.qml

```diff
@@ -0,0 +1,176 @@
+import QtQuick 2.11
+import QtQuick.Shapes 1.11
+
+import QgsQuick 0.1 as QgsQuick
+
+Item {
+  id: highlight
+
+  // color for line geometries
+  property color lineColor: "black"
+  // width for line geometries
+  property real lineWidth: 4
```

Comment:
Small note about default values - even if this and properties below (related to any width/height) overwritten in main.qml while init the component, maybe consider adding QgsQuick.Utils.dp or remove default values (hmm, maybe could be interesting when extracted to similar app)?

Reply:
fixed in 6a50bbf